okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -150,12 +151,16 @@ private static ResponseBody buffer(MediaType mediaType, byte[] bodyBytes) {
 
     @Override
     public void enqueue(Callback callback) {
+        DeferredTracer tracer = new DeferredTracer();
         ListenableFuture<Limiter.Listener> limiterListener = limiter.acquire();
-        request().tag(ConcurrencyLimiterListener.class).limiterListener().setFuture(limiterListener);
+        request().tag(ConcurrencyLimiterListener.class).setLimiterListener(limiterListener);
         Futures.addCallback(limiterListener, new FutureCallback<Limiter.Listener>() {
             @Override
             public void onSuccess(Limiter.Listener listener) {
-                enqueueInternal(callback);
+                tracer.withTrace(() -> {
```

Comment:
:+1: good spot! Would be a shame to regress here - could you change one of the tests to capture this desired behaviour?

Reply:
done